ingestion/tests/cli_e2e/test_cli_db_base_common.py

```diff
@@ -0,0 +1,162 @@
+#  Copyright 2022 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Test database connectors which extend from `CommonDbSourceService` with CLI
+"""
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+from sqlalchemy.engine import Engine
+
+from metadata.ingestion.api.sink import SinkStatus
+from metadata.ingestion.api.source import SourceStatus
+from metadata.ingestion.api.workflow import Workflow
+
+from .test_cli_db_base import PATH_TO_RESOURCES, CliDBBase
+
+
+class CliCommonDB(object):
```

Comment:
let's kill the `object` inheritance here, not needed at all

Reply:
Also maybe silly question, I see that we are creating classes as `class MysqlCliTest(CliCommonDB.TestSuite):`. What's the benefit of wrapping the class with `CliCommonDB`? To not trigger the tests?

Maybe we can then have a function returning the `TestSuite` or not use `test_` as a module name 🤔

Reply:
Using this approach we can create an abstract `TestCase` class that avoids issues with test runner discovery and let us still import the abstract test from another module. I had already used this approach in the past for the Secrets Manager tests.

Reply:
cool thanks for explaining. Let's then just remove the `object` notation and call it a day!